### PR TITLE
Refactored EntityGanttTest POM :

### PR DESCRIPTION
**Pull request by guram21**
https://trello.com/c/4wpBFByl
- Applied method autoFillOutForm
- Added methods getRowValues, getTitle to EntityBasePage
- Relocated variables NO_RECORDS, RECORD_NOT_DELETED, DRAFT_NOT_DELETED to EntityGanttTest

### CODE REVIEW

File: src/test/java/model/base/EntityBasePage.java [hunk 1]
```diff
@@ -93,6 +94,19 @@ public String getRowIconClass(int rowNumber) {
...
+        List<WebElement> findElements = getRows().get(rowNumber).findElements(By.tagName("td"));
+        for (int i = 1; i < findElements.size() - 1; i++) {
+            if (!findElements.get(i).getText().equals("")) {
+                WebElement webElement = findElements.get(i);
+                String text = webElement.getText();
+                list.add(text);
+            }
+        }
+        return list;
+    }
```
Comment (MaxBurdin) [inline src/test/java/model/base/EntityBasePage.java:108]: There is getAllRows method in the same class which gets all values of a record. I think we need to keep just one of them.

File: src/test/java/test/entity/EntityGanttTest.java [hunk 2]
```diff
@@ -5,155 +5,121 @@
...
-    @Test(dependsOnMethods = "editRecordGanttTest")
-    public void deleteRecordGanttTest() {
-
-        GanttTablePage ganttTablePage = new MainPage(getDriver()).clickMenuGantt().clickListButton();
+    @Test(dependsOnMethods = "editRecordTest")
+    public void deleteRecordTest() {
+        GanttTablePage ganttTablePage = new MainPage(getDriver())
+                .clickMenuGantt()
+                .clickListButton();
         final String recordTitle = ganttTablePage.getTitle(0);
```
Comment (MaxBurdin) [inline src/test/java/test/entity/EntityGanttTest.java:75]: Names representing constants (final variables) must be all uppercase using underscore to separate words

File: src/test/java/test/entity/EntityGanttTest.java [hunk 3]
```diff
@@ -5,155 +5,121 @@
...
-    @Test(dependsOnMethods = "createDraftGanttTest")
-    public void deleteDraftGanttTest() {
-
-        GanttTablePage ganttTablePage = new MainPage(getDriver()).clickMenuGantt().clickListButton();
+    @Test(dependsOnMethods = "createDraftTest")
+    public void deleteDraftTest() {
+        GanttTablePage ganttTablePage = new MainPage(getDriver())
+                .clickMenuGantt()
+                .clickListButton();
         final String entityTitle = ganttTablePage.getTitle(0);
```
Comment (MaxBurdin) [inline src/test/java/test/entity/EntityGanttTest.java:106]: Names representing constants (final variables) must be all uppercase using underscore to separate words

**Review by MaxBurdin — CHANGES_REQUESTED** [review]

File: src/test/java/model/base/EntityBasePage.java (hunk 1 above)
Comment (oooleks) [reply src/test/java/model/base/EntityBasePage.java:108]: Yes, we've got `List<String> getRow(int rowNumber) `method but it does not collect User value and does not skip empty values like this one. Let's Guram confirm

Comment (guram21) [reply src/test/java/model/base/EntityBasePage.java:108]: As we agreed with Max, going to use his method:
```
    @FindBy (xpath = "//td[9]")
    private WebElement user;

public List<String> getAllRows(int rowNumber) {
    List<String> lst = getRow(rowNumber, "//td/a");
    lst.add(user.getText());
    return lst;
}
```

**Review by oooleks — APPROVED** [review]

**Review by MaxBurdin — APPROVED** [review]

**Review by GalinaRuban — APPROVED** [review]

**Pull request merged**